Dockerfile

```diff
@@ -0,0 +1,15 @@
+# use Docker's provided python image
+FROM python:2.7
+MAINTAINER natebrennand <natebrennand@gmail.com>
+
+# install all packages
+COPY ./config/requirements.txt /requirements.txt
+RUN pip install -r /requirements.txt
+
+# add the application directories
+ADD ./density /density
+WORKDIR /density
+
+# expose the port and start the server
+EXPOSE 5000
+CMD gunicorn density
```

Comment:
We'll add the gunicorn configurations but we should think more about what the exact logging infrastructure is.
We can either read from the stdout of docker or mount a "volume" and write the log file to that.